src/sa_web/static/js/routes.js

```diff
@@ -7,6 +7,11 @@ var Shareabouts = Shareabouts || {};
     routes: {
       '': 'viewMap',
       'place/new': 'newPlace',
+      'place/new/observation': 'newPlaceObservation',
+      'place/new/question': 'newPlaceQuestion',
+      'place/new/idea': 'newPlaceIdea',
+      'place/new/complaint': 'newPlaceComplaint',
+      'place/new/greenwall': 'newPlaceGreenwall',
```

Comment:
It would be great to have a user define new routes in their config, but I'm not sure about the specifics to make that work. Actually, I think we can abstract this to work in the general code by passing the new dynamic place type as a function param ie `'place/new/:place_type': 'newDynamicPlace',`. 

Not sure if we have time now to make this refactor before the event, so I just pushed a commit that abstracts the dynamic form logic into the duwamish flavor code.